getting-started/openshift/installation.md

```diff
@@ -10,7 +10,7 @@ Install an OpenShift v4 cluster with {{site.prodname}}.
 
 ### Value
 
-Augments the applicable steps in the [OpenShift documentation](https://cloud.redhat.com/openshift/install)
+Augments the applicable steps in the {% include open-new-window.html text='OpenShift documentation' url='https://cloud.redhat.com/openshift/install' %}
```

Comment:
I wonder if this should link to: https://docs.openshift.com/. (The existing link asks me to sign into redhat, so I can't tell what content it actually ends up with.)

Reply:
One of our regular openshift users in eng might know.